internal/extensions/global_store_paths.go

```diff
@@ -21,7 +21,7 @@ func getExtensionsCachePath(globalStore *global.GlobalStore) string {
 }
 
 func ensureExists(dirPath string) error {
-	return os.MkdirAll(dirPath, 755)
+	return os.MkdirAll(dirPath, 0o755)
```

Comment:
I do not understand this change

Reply:
The filesystem uses octal permissions as such 755 is not the same as 0o755